apps/aecore/src/aec_consensus_hc.erl

```diff
@@ -555,6 +556,10 @@ default_pinning_behavior() ->
     Fun = fun() -> get_consensus_config_key([<<"default_pinning_behavior">>], false) end,
     aeu_ets_cache:get(?ETS_CACHE_TABLE, default_pinning_behavior, Fun).
 
+hyperchain_static_coinbase() ->
+    Fun = fun() -> get_consensus_config_key([<<"hyperchain_static_coinbase">>], false) end,
```

Comment:
```suggestion
    Fun = fun() -> get_consensus_config_key([<<"hyperchain_static_coinbase">>], 0) end,
```

Reply:
I don't remember how it works but `false` is definitely wrong, better with `0`

Reply:
But, I'll make that change, if my fix for default values from schema _is_ in, then that will sort itself out.